src/main/java/blusunrize/immersiveengineering/common/blocks/IEBaseBlock.java

```diff
@@ -302,33 +316,29 @@ public IELadderBlock(String name, Block.Properties material,
 			super(name, material, itemBlock, additionalProperties);
 		}
 
+		//TODO review: actually not sure if this override is still needed
+		// 			  	It appears to handle the moment when players jump
+		//				  on a ladder, preventing fall damage, and limit
+		//				  the motion accordingly. This should be already
+		//				  the vanilla default by now.
```

Comment:
I think you're right, we probably need to override something like `isLadder` then. `IELadderBlock` can probably be removed in that case.

Reply:
Added `isLadder()` override, also added notes in the wood/metal ladder blocks, which may be better inherited directly from `mc::LadderBlock`. I am not entirely sure yet because I don't know what additional meaning these ladders have when forming multi blocks.